featuretools/computational_backends/utils.py

```diff
@@ -213,20 +213,22 @@ def get_client_cluster():
     """
     Separated out the imports to make it easier to mock during testing
     """
-    from distributed import Client, LocalCluster
+    distributed = import_or_none("distributed")
+    Client = distributed.Client
+    LocalCluster = distributed.LocalCluster
 
     return Client, LocalCluster
 
 
 def _validate_cutoff_time(
-    cutoff_time: typing.Union[dd.DataFrame, pd.DataFrame, str, datetime],
+    cutoff_time: typing.Union[pd.DataFrame, str, datetime],
```

Comment:
Given we handle the Dask case on L231, is `dd.DataFrame` still a valid type for `cutoff_time`?

Reply:
Technically, yes, I think `dd.DataFrame` is still valid type as input to this function, but it will always get converted to a pandas dataframe before it gets used. We get an error here if Dask isn't installed though because `dd` becomes `None` and `None` doesn't have a `DataFrame` attribute.

If there is an easy way to fix that, I can update, but wasn't sure off the type of my head.